os/customizing-sshd.md

```diff
@@ -41,38 +41,73 @@ systemd:
       enable: true
       contents: |
         [Socket]
-        ListenStream=2222
+        ListenStream=222
```

Comment:
See the commit message https://github.com/coreos/docs/pull/1100/commits/3c184a0d0c1db8eb3e220db1fe8466b75135f794